### PR TITLE
Auctions Overview Design review pt 2

### PR DESCRIPTION
**Pull request by orta**
Fixes - #1209 #1207 #1205

Does the UI aspect of #1206 - the initial tap part is gonna require a deeper dive that I'm gonna punt on until later.

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Warning</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARArtworkFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkFavoritesNetworkModelSpec</a> in <code>test_getFavorites__makes_request_if_no_request_is_in_progress</code></li>
<li><a href="https://github.com/search?q=ARArtworkViewControllerBuyButton+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkViewControllerBuyButtonSpec</a> in <code>test_buy_button__posts_order_if_artwork_has_no_edition_sets</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__passes_artwork_to_share_controller</code></li>
<li><a href="https://github.com/search?q=ARFairViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARTopMenuViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARTopMenuViewControllerSpec</a> in <code>test_navigation__favorites__logged_in__tab_behavior__already_selected__pops_to_root</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Artsy/Views/Auction/AuctionBannerView.swift [hunk 1]
```diff
@@ -48,28 +48,41 @@ extension AuctionBannerView {
...
+                if let _ = self.superview {
+                    $0.startTimer()
+                }
+            }
+            self.countdownView = countdownView
+            self.addSubview(countdownView)
+        }
+
         logoImageView.constrainHeight("70")
+        constrainHeight("200")
```
Comment (orta) [inline Artsy/Views/Auction/AuctionBannerView.swift]: the old version ( iPhone would be done dynamically ) would only work if a countdown view existed, but as there is a closed state now without one, then it's simpler to just have a standard size. This affects the snapshots for all the phone versions.

File: Artsy/Views/Auction/AuctionBannerView.swift [hunk 2]
```diff
@@ -48,28 +48,41 @@ extension AuctionBannerView {
...
+            let countdownView = ARCountdownView(color: .whiteColor()).then {
+                let model = self.viewModel
+                let isNotOpenYes = model.saleAvailability == .NotYetOpen
+
+                $0.targetDate = isNotOpenYes ? model.startDate : model.closingDate
+                $0.heading = isNotOpenYes ? "Opening In" : "Closing In"
+
+                if let _ = self.superview {
+                    $0.startTimer()
+                }
```
Comment (alloy) [inline Artsy/Views/Auction/AuctionBannerView.swift]: I don’t think that `then { … }` makes things more readable when it spans over so many lines tbh

File: Artsy/Views/Collection_View_Cells/ARSaleArtworkFlowCollectionViewCell.m [hunk 3]
```diff
@@ -43,7 +43,9 @@ - (void)createSubviews
     self.artworkImageView.clipsToBounds = YES;
     [self.contentView addSubview:self.artworkImageView];
 
-    UIColor *darkGrey = [UIColor colorWithHex:0x666666];
+    // HACK: This *should* be 0x666666 but once it gets to the iPhone it's 0x535353
+    // so offset that, by making adding that difference to the original.
+    UIColor *darkGrey = [UIColor colorWithHex:0x797979];
```
Comment (alloy) [inline Artsy/Views/Collection_View_Cells/ARSaleArtworkFlowCollectionViewCell.m]: This should really be figured out at some point.

**Pull request merged**